src/renderer/components/dock/logs/download-logs-dropdown.module.scss

```diff
@@ -0,0 +1,37 @@
+.dropdown {
```

Comment:
Styles for reusable dropdown?

Reply:
Depends on what do you mean. This is is css-module so the class name would be unique.
If you're about customization of component I would add some more `--css-variables` to manage most important parts.

Reply:
I mean that is it something that could be used for re-usable dropdown component that I mentioned on other comments. I know that this is CSS modules and and the class names wont collide.

Reply:
Created a `<Dropdown/>` component but without any styles provided. It's better to keep components "headless" to avoid CSS specificity issues and overwrite style attempts in the future.